src/collections/src/NICollectionViewModel.h

```diff
@@ -45,8 +45,6 @@
 
 #pragma mark Creating Collection View Models
 
-// Designated initializer.
-- (id)initWithDelegate:(id<NICollectionViewModelDelegate>)delegate;
 - (id)initWithListArray:(NSArray *)listArray delegate:(id<NICollectionViewModelDelegate>)delegate;
```

Comment:
Maybe we should remove the initializers from the protocol altogether?

Reply:
Updated.